cstar/args.py

```diff
@@ -106,10 +109,14 @@ def add_cstarpar_arguments(parser):
     _add_common_arguments(parser)
     _add_strategy_arguments(parser)
     _add_ssh_arguments(parser)
+    _add_jmx_auth_arguments(parser)
     parser.add_argument('command', help='Command to run once for each Cassandra host')
 
 def _add_ssh_arguments(parser):
     parser.add_argument('--ssh-username', help='Username for ssh connection', default=None)
     parser.add_argument('--ssh-password', help='Password for ssh connection', default=None)
     parser.add_argument('--ssh-identity-file', help='Identity file for ssh connection', default=None)
-    
\ No newline at end of file
+
+def _add_jmx_auth_arguments(parser):
+    parser.add_argument('--jmx-username', help='JMX username', default=None)
+    parser.add_argument('--jmx-password', help='JMX password', default=None)
```

Comment:
I think we might not want to pass the jmx password as an argument but rather prompt the user to enter it if `--jmx-username` is set. This could avoid having the password showing up in the `ps aux` output of any connected user.
I admit any `nodetool` command sent with cstar on such a cluster would require to put it in the command nonetheless, but ¯\_(ツ)_/¯

I also think the password shouldn't be stored in the `job.json` file. This would require to prompt again when we resume/continue a job. This will require to blacklist the jmx password in the [jobwriter](https://github.com/spotify/cstar/blob/65cbb49f52/cstar/jobwriter.py) and detect [here](https://github.com/spotify/cstar/blob/65cbb49f52/cstar/cstarcli.py#L50-L63) that we have a jmx username and ask again for the password.